test/api/controllers/authController.test.js

```diff
@@ -285,15 +284,21 @@ describe('Test the auth controller', () => {
           {message: 'You must be authenticated to execute that action'});
     });
 
-    it('should throw if the token has already been refreshed', () => {
-      return should(() => authController.refreshToken(new Request(
-        {},
-        {
-          token: {userId: 'foo', _id: 'bar', refreshed: true},
-          user: {_id: 'bar'}
-        }
-      )))
-        .throw(UnauthorizedError, {message: 'Invalid token'});
+    it('should throw if the token has already been refreshed', done => {
+      authController.throw = sinon.spy();
+
+      try {
+        authController.refreshToken(new Request(
+          {},
+          {
+            token: { userId: 'foo', _id: 'bar', refreshed: true },
+            user: { _id: 'bar' }
+          }
+        ));
+        done(authController.throw('invalid_token'));
+      } catch (e) {
+        done(e);
+      }
```

Comment:
This test always passes.

Reply:
Still not good (here and other cases)
The test will pass in both cases where `refreshToken` throws or not. 
Actually, here, as `throw` is overriden to a `spy`, the `catch` clause is never executed.

possible implementation:
```javascript 
    it('should throw if the token has already been refreshed', () => {
      authController.throw = sinon.spy();

      authController.refreshToken(new Request(
        {},
        {
          token: { userId: 'foo', _id: 'bar', refreshed: true },
          user: { _id: 'bar' }
        }
      ));
      should(authController.throw).be.calledWith('invalid_token');
    });
```
Or, by actually throwing:

```javascript
it('should throw', () => {
  return should(authController.refreshToken(request))
    .be.rejectedWith(ExpectedError, {message: 'some error'});
});
```